cogs/voice_channels.py

```diff
@@ -1,9 +1,13 @@
 import discord
 import re
 from discord.ext import commands
+from discord.voice_client import opus
+import os
+from pytube import YouTube
```

Comment:
This adds a dependency to the project that wasn't there before, in the future these should be announced.